MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,572 @@
+---
+layout: post
+title: Toolbar Customization in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about toolbar customization support in the Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Toolbar Customization in .NET MAUI Image Editor (SfImageEditor)
+
+You can customize the existing toolbar appearance, visibility, position, and orientation of each toolbar and can include new toolbars.
+
+## Toolbar style customization
+
+To customize the appearance of the toolbar by using the `ToolbarSettings` property. Below are some of the properties you can customize:
+
+* `Background`: Specifies background color of the toolbar.
+* `IconColor`: Specifies the color of the icons in the toolbar.
+* `IconSize`: Specifies the size of the icons in the toolbar. The default value is 48.
+* `DisabledColor`: Specifies the color of the disabled icons in the toolbar.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="image.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings Background="LightGreen"
+                                                IconColor="Tomato"
+                                                DisabledColor="Lavender"
+                                                IconSize="50, 50" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+imageEditor.ToolbarSettings.Background = Brush.LightGreen;
+imageEditor.ToolbarSettings.IconColor = Colors.Tomato;
+imageEditor.ToolbarSettings.DisabledColor = Colors.Lavender;
+imageEditor.ToolbarSettings.IconSize = new Size(50, 50);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar Orientation
+
+The toolbar orientation can be changed to achieve the desired design by using the `Orientation` property in the `ImageEditorToolbar` class. The default value is `ToolbarOrientation.Horizontal`.
+
+For example, you can set the toolbar to a vertical orientation:
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+headerToolbar.Orientaion = ToolbarOrientation.Vertical;
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar Position
+
+To adjust the alignment of the toolbar by using the `Position` property in the `ImageEditorToolbar` class. The default value is `ToolbarPosition.Start`.
```

Comment:
The toolbar can be positioned before or after the image of the ImageEditor by using the `Position` property in the `ImageEditorToolbar` class. The default value is `ToolbarPosition.Start`.

Reply:
Updated